inc/glpi.class.php

```diff
@@ -82,9 +82,9 @@ public function initLogger() {
          }
          $logger->pushHandler($handler);
          $this->loggers[$type] = $logger;
-         $var = strtoupper($type . 'logger');
-         $$var = $this->loggers[$type];
       }
+      $PHPLOGGER = $this->loggers['php'];
```

Comment:
can you explain this one ?

Reply:
This replace the `$$var`, to prevent `$PHPLOGGER` and `$SQLLOGGER` to be detected as not used from the CI tool. Anther way would maybe be to tell it to ignore warnings on those ones.